.github/workflows/test-sequencer.yml

```diff
@@ -0,0 +1,73 @@
+name: Test sequencer
+
+on:
+  workflow_dispatch:
```

Comment:
Can we also schedule this to run once a week automatically?

Reply:
Done